src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -282,7 +283,10 @@ def validate_encryption(namespace):
     ''' Builds up the encryption object for storage account operations based on the
     list of services passed in. '''
     if namespace.encryption:
-        from azure.mgmt.storage.models import Encryption, EncryptionServices, EncryptionService
+        rt = ResourceType.MGMT_STORAGE_STORAGE_ACCOUNTS
+        Encryption = get_versioned_models(rt, model='Encryption')
+        EncryptionServices = get_versioned_models(rt, model='EncryptionServices')
+        EncryptionService = get_versioned_models(rt, model='EncryptionService')
```

Comment:
Here again, the original is clearly superior to the new.  Why not something like:

```Python
Encryption, EncryptionServices, EncryptionService = versioned_models(ResourceType.MGMT_STORAGE_STORAGE_ACCOUNTS, 'Encryption', 'EncryptionServices', 'EncryptionService')
```

This isn't great, but it's closer to the "easy" way and is easily implemented with minor changes. In reality, I would prefer it look like:
```Python
Encryption, EncryptionServices, EncryptionService = versioned_models(ResourceType.MGMT_STORAGE_STORAGE_ACCOUNTS)
```

So that it is, for all intents and purposes, functionally identical to the original while still handling the version stuff, but a solution does not immediately spring to mind for how to accomplish this--however, doing this would solve the ugly syntax of importing the enum for the choice list as well.